IgdbApiWrapper.js

```diff
@@ -13,9 +14,77 @@ function IgdbWrapper(){
 };
 
 IgdbWrapper.prototype.getGamesData = async function(){
-    let response = await request.get(this.options);
+    const gamesEndpointOptions = this.makeEndpointOptions({
+        path: 'games/',
+        queryParams: {
+            fields: ['name', 'popularity', 'cover.url', 'id'],
+            order: 'popularity:desc'    
+        }
+      }
+    );
+    let response = await request.get(gamesEndpointOptions);
     let gamesData = JSON.parse(response);
     return gamesData;    
 };
+IgdbWrapper.prototype.getGameData = async function(gameId){
+    const gameEndpointOptions = this.makeEndpointOptions({
+        path: `${"games/" + gameId}`,
+        queryParams: {
+            fields: ['name', 'summary', 'first_release_date', 'platforms',
+                'developers', 'websites', 'popularity', 'cover.cloudinary_id']
+        }
+    });
+    let response = await request.get(gameEndpointOptions);
+    let gameData = JSON.parse(response);
+    return gameData;
+};
+IgdbWrapper.prototype.makeEndpointOptions = function(endpointParams){
+    let endpointOptions = Object.assign({}, this.options);
+    endpointOptions.url = buildUrl(this.options.url, endpointParams);
+    return endpointOptions;
+};
+IgdbWrapper.prototype.getPlatforms = async function(platforms){
+    const platformEndpointOptions = this.makeEndpointOptions({
+        path: `${"platforms/" + platforms.join(",")}`,
+        queryParams: {
+            fields: ['name'],   
+            }
+        }
+    );
+    let response = await request.get(platformEndpointOptions);
+    return JSON.parse(response);
+};
+IgdbWrapper.prototype.getWebsites = function(websites){
+    const categories = {
```

Comment:
Move this categories object out of the function. You can keep it at the top level of the file for now or move all constants into separate module.
Also, start constant enumerations with capitalized letter, for ex. `Categories`.